libr/core/cmd_anal.c

```diff
@@ -6826,7 +6826,7 @@ static void rowlog(RCore *core, const char *str) {
 	}
 }
 
-static void rowlog_done(RCore *core) {
+R_API void rowlog_done(RCore *core) {
```

Comment:
dont make non r_* apis public with R_API, this should be static, or refactored into r_print